### PR TITLE
Batch edit modal

### PR DESCRIPTION
**Pull request by xIrusux**
## Changes in this pull request
Resolves #

### CODE REVIEW

File: assets/js/src/core/components/modal/footer/modal-footer.tsx [hunk 1]
```diff
@@ -13,22 +13,31 @@
 
 import React from 'react'
 import { useStyle } from '@Pimcore/components/modal/footer/modal-footer.styles'
+import { Flex } from '@Pimcore/components/flex/flex'
 
 interface IModalFooterProps {
   children?: React.ReactNode
   buttonAlignment?: 'space-between' | 'end'
```
Comment (vin0401) [inline assets/js/src/core/components/modal/footer/modal-footer.tsx]: I think we can improve this a bit more. I think we used that before we had our flex component.
We could define the justify prop from flex as prop here.

**Review by vin0401 — CHANGES_REQUESTED** [review]
We are on the right way. Just one small thing, I would like to improve. :)

File: assets/js/src/core/components/modal/footer/modal-footer.tsx (hunk 1 above)
Comment (xIrusux) [reply assets/js/src/core/components/modal/footer/modal-footer.tsx:20]: Ah! I was meant to be using the shared component flex container! 
Good spot :)

Comment (xIrusux) [reply assets/js/src/core/components/modal/footer/modal-footer.tsx]: @vin0401 fixed :)

**Review by vin0401 — APPROVED** [review]

**Pull request merged**